packages/contracts/test/SphinxTestUtils.sol

```diff
@@ -147,7 +156,7 @@ contract SphinxTestUtils is SphinxConstants, StdCheatsSafe, SphinxUtils, SphinxI
             string(abi.encodePacked(key, ".arguments"))
         );
         string[] memory arguments = argumentsBytes.length == 32 &&
-            bytes32(argumentsBytes) == bytes32(0)
+            toBytes32(argumentsBytes) == bytes32(0)
```

Comment:
We have to use the above `toBytes32` function to workaround a limitation where bytes cannot be implicitly converted to bytes32. This wasn't an issue previously because were were compiling with `0.8.15`. But now we're compiling everything with `0.8.4`, so we have to implement a workaround. 

Originally, I was just going to have the core contracts compile with `0.8.4` and then compile everything else with `0.8.15` which would have resolved that, but I found things didn't immediately work that way and it seemed relatively insignificant to just implement this workaround instead.

Reply:
Sounds good. Could you include that in the docs for `toBytes32`?